routes/product.js

```diff
@@ -7,6 +7,8 @@ const router = express.Router();
 
 router.post('/', createProduct);
 
+router.get('/', getProducts);
```

Comment:
no need for an empty space between route handlers, this is usually how i structure it:

```
const router = express.Router();

router.post();
router.get();
router....
router...

export default router;
```